### PR TITLE
UNOMI-435 Migrate existing events scope to source

### PR DESCRIPTION
**Pull request by anatol-sialitski**
This PR contains the migration from v1.5.0 to v2.0.0 which updates mapping for an index with prefix `context-event`. Adds the `sourceId` field and copies value from the `scope` field to it.

### CODE REVIEW

File: tools/shell-commands/src/main/java/org/apache/unomi/shell/migration/impl/MigrationTo200.java [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+
+    private void updateMapping(final String indexName, final CloseableHttpClient httpClient) throws IOException {
+        HttpPut httpPut = new HttpPut(esAddress + "/" + indexName + "/_mapping");
+
+        httpPut.addHeader("Accept", "application/json");
+        httpPut.addHeader("Content-Type", "application/json");
+
+        String request = "{\n" +
+                "\"properties\": {\n" +
+                " \"sourceId\": {\n" +
```
Comment (sergehuber) [inline tools/shell-commands/src/main/java/org/apache/unomi/shell/migration/impl/MigrationTo200.java:94]: Seems like the analyzer is missing here.

**Review by sergehuber — CHANGES_REQUESTED** [review]
Hello @anatol-sialitski thanks for the PR. I found a small issue should be easy to fix.

File: tools/shell-commands/src/main/java/org/apache/unomi/shell/migration/impl/MigrationTo200.java (hunk 1 above)
Comment (anatol-sialitski) [reply tools/shell-commands/src/main/java/org/apache/unomi/shell/migration/impl/MigrationTo200.java:94]: fixed

**Review by sergehuber — DISMISSED** [review]
Thanks for adding the analyzer so quickly ! I'll merge it.

**Comment by sergehuber** [comment]
Oops I spoke too fast it seems there is a problem with the integration tests, could you check it please ?

**Comment by anatol-sialitski** [comment]
@sergehuber 
IT tests failed for Java 11, but work for Java 8. Looks like delay is too short to process an event and to refresh index during tests running.

**Review by sergehuber — APPROVED** [review]
Ok looks good to me will merge and test again

**Pull request merged**